src/node/node_state.h

```diff
@@ -1320,26 +1320,6 @@ namespace ccf
 
     void setup_basic_hooks()
     {
-      // When a transaction that changes the configuration commits globally,
-      // inform the host of any nodes that no longer need to be tracked.
-      network.nodes.set_global_hook(
```

Comment:
This global hook is now gone. The channel (and associated host outgoing connection) will get closed by Raft directly on global commit of the configuration in which the node is deleted.